pom.xml

```diff
@@ -166,5 +166,71 @@
         <module>drools-examples-android</module>
       </modules>
     </profile>
+    <profile>
+      <id>run-code-coverage</id>
+      <properties>
+        <jacoco.excludes></jacoco.excludes>
+        <jacoco.exec.file>${project.build.directory}/jacoco.exec</jacoco.exec.file>
```

Comment:
Let's take the jacoco.exec.file from kie-parent, which resides in the project's root directory/target, to fully benefit from the append mode and get cross-module coverage.